tests/test_gate_environment.py

```
@@ -59,3 +75,58 @@ def test_gateway_and_oauth_modules_use_gateway_paths_helper():
+import mcp_gateway
+
+async def main():
+    await mcp_gateway.proxy_manager.refresh()
+    print(json.dumps({
+        "config_path": str(mcp_gateway.proxy_manager.registry.config_path),
+        "error": mcp_gateway.proxy_manager.registry._last_refresh_error,
+    }))
+
+asyncio.run(main())
+"""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)